src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpOverallOutcomeController.cs

```diff
@@ -0,0 +1,239 @@
+                    if (application.GatewayReviewStatus == GatewayReviewStatus.Fail)
+                        return RedirectToAction("ApplicationUnsuccessful", new {applicationId});
+                    return RedirectToAction("ApplicationUnsuccessful",  new {applicationId});
+                case ApplicationStatus.FeedbackAdded:
+                    return RedirectToAction("FeedbackAdded", new {applicationId});
+                case ApplicationStatus.Withdrawn:
+                    return RedirectToAction("ApplicationWithdrawn", new {applicationId});
+                case ApplicationStatus.Removed:
+                    return RedirectToAction("ApplicationRemoved",  new {applicationId});
+                case ApplicationStatus.GatewayAssessed:
+                    if (application.GatewayReviewStatus == GatewayReviewStatus.Reject)
+                        return RedirectToAction("ApplicationRejected", 
+                            new {applicationId});
+                    return RedirectToAction("ApplicationSubmitted",  new {applicationId});
+                case ApplicationStatus.Submitted:
+                case ApplicationStatus.Resubmitted:
+                    return RedirectToAction("ApplicationSubmitted",  new {applicationId});
+                default:
+                    return RedirectToAction("TaskList", "RoatpApplication", new {applicationId});
+            }
+        }
+
+        [HttpGet]
+        [Route("ApplicationStatus")]
+        [Authorize(Policy = "AccessApplication")]
+        public async Task<IActionResult> ApplicationUnsuccessful(Guid applicationId)
+        {
+            var application = await _apiClient.GetApplication(applicationId);
+            var applicationData = application.ApplyData.ApplyDetails;
+
+            var oversightReview = await _apiClient.GetOversightReview(applicationId);
+            
+
+            // this will change based on coming stories and overturns etc
+            var applicationGatewayReviewStatusFail = (application.GatewayReviewStatus == GatewayReviewStatus.Fail);
+
+            if (applicationGatewayReviewStatusFail)
+            {
+                var unsuccessfulModel = await BuildApplicationSummaryViewModel(applicationId);
+                return View("~/Views/Roatp/ApplicationUnsuccessful.cshtml", unsuccessfulModel);
+            }
+
+            var applicationUnsuccessfulModerationFail = false;
+            if (application?.GatewayReviewStatus == GatewayAnswerStatus.Pass)
+            {
+                if (application?.ModerationStatus != null
+                    && application?.ModerationStatus == ModerationStatus.Fail
+                    && oversightReview.ModerationApproved.HasValue
+                    && oversightReview.ModerationApproved == true)
+                {
+                    applicationUnsuccessfulModerationFail = true;
+                }
+            }
+
+            var model = new ApplicationSummaryWithModeratorDetailsViewModel
```

Comment:
Something to do at a later time but might be able to reuse BuildApplicationSummaryViewModel

Reply:
this is based on BuildSummaryViewModel, and it's not clear at the moment whether we decorate children of BASVM for each story or major part of the outcome work, or whether BSVN will just be a big old model we re-use.  I split it for now, but I think as we do outcome stories, it will become clearer how to proceed